src/main/java/step1/main.java

```diff
@@ -0,0 +1,11 @@
+package step1;
+
+public class main {
```

Comment:
클래스의 이름은 대문자로 시작하는 것이 컨벤션입니다 :)

```suggestion
public class Main {
```